blog/2024-01-24-embed-figma/index.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Embed Figma in Stately
+description: You can now embed Figma frames in Stately states
+tags: [stately, studio, figma, design, statecharts, state machines, xstate]
+authors: [nick]
+date: 2024-01-24
+slug: 2024-01-24-embed-figma
+image: /blog/2024-01-24-embed-figma/figma-dark.png
+---
+
+We already kicked off 2024 with several major updates to Stately such as inspecting live XState code, syncing with GitHub, and writing code for actions and actors in the Studio. Now we’re excited to announce that we've also launched an integration with Figma.
```

Comment:
```suggestion
We already kicked off 2024 with several major updates to Stately such as inspecting live XState code, syncing with GitHub, and writing code for actions and actors in the Studio. Now we’re excited to announce that we've also launched an integration with Figma!
```
Just because "we're excited" but not a must.

Reply:
On one hand it make sense to add links to "inspecting XState code", "syncing with GitHub". It also could be helpful to refer to these by their "official" feature names. On the other hand, maybe we don't need links in the opening paragraph before the truncation. Curious to hear what you or others think about links in the opening paragraph.